MPF.Core/Modules/DiscImageCreator/Parameters.cs

```diff
@@ -496,7 +496,18 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
                             case RedumpSystem.SonyPlayStation3:
                             case RedumpSystem.SonyPlayStation4:
                             case RedumpSystem.SonyPlayStation5:
-                                trimLength = 264;
+                                if (info.SizeAndChecksums!.Layerbreak3 != default)
```

Comment:
Does this need to be done for either Aaru or Redumper as well? I'd expect Redumper may benefit from at least part of this.

Reply:
Sorry, but I can't understand the question.
I do not know much about Aaru.

Reply:
@mnadareski this PR already does this for Redumper.
and Aaru/Parameters.cs does not currently parse Layerbreak for blurays anyway.

Reply:
I paid attention, I swear. That was what I was looking for and then promptly overlooked it.